netci.groovy

```diff
@@ -0,0 +1,26 @@
+import jobs.generation.Utilities
+
+def project = GithubProject
+def branch = GithubBranchName
+def isPR = true
+def distroList = ['alpine', 'centos/6', 'centos/7', 'debian', 'fedora', 'opensuse', 'ubuntu/14', 'ubuntu/16', 'ubuntu/17']
+
+distroList.each { distro ->
+    def(distroName, distroVersion) = distro.tokenize('/')
+    if (distroVersion != null) {
+        distroName = distroName.concat("_" + distroVersion)
+    }
```

Comment:
The logic to replace `/` with `_` seems overly complicated.  Why not simply call replace?  `distro = distro.replace('/', '_')

Reply:
Yes, `distro = distro.replace('/', '_')` works too.

Reply:
It is more than about functionality though - it is readability and clarity.  The current pattern complicates things and almost makes the reader think there is more to what is really being done.

Reply:
Resolved through https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/14